neos/neos-ui
# !!! FEATURE: Separate properties and references in clientEval

**What I did**
As discussed in https://github.com/neos/neos-development-collection/pull/5156 references should be available separately in ClientEval 

**How I did it**

I adjusted the logic for loading data for the right sidebar to be separated into properties and references. 

depends on https://github.com/neos/neos-ui/pull/3810  

**How to verify it**

Use ClientEval:console.log(node) in the right to see that references are no longer in the properties array but now their separate item: 
<img width="1082" alt="Bildschirmfoto 2024-06-24 um 18 01 14" src="https://github.com/neos/neos-ui/assets/91674611/125c4dd1-c926-4cb1-a256-6a8d6ebd4446">
